windows/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+target_sources(ninja PRIVATE 
+    ninja.manifest
+    ninja.rc.in
+)
+
+string(TIMESTAMP NINJA_CURRENT_YEAR "%Y")
+
+configure_file(ninja.rc.in ninja.rc @ONLY)
+
+target_sources(ninja PRIVATE
+    ${CMAKE_CURRENT_BINARY_DIR}/ninja.rc
+)
```

Comment:
For what it's worth, I don't think it makes sense to create a subdirectory just for windows. This is only 12 lines of cmake code, it should just go into the main cmakelists.txt